src/joystick/controller_type.c

```diff
@@ -109,7 +109,7 @@ EControllerType GuessControllerType( int nVID, int nPID )
 		return k_eControllerType_UnknownNonSteamController;
 	}
 
-	for ( iIndex = 0; iIndex < sizeof( arrControllers ) / sizeof( arrControllers[0] ); ++iIndex )
+	for ( iIndex = 0; iIndex < SDL_arraysize(arrControllers); ++iIndex )
```

Comment:
Please back out changes to this file. It's shared with external code and should not be touched.